lib/routific.rb

```diff
@@ -9,7 +9,14 @@
 
 # Main class of this gem
 class Routific
-  attr_reader :token, :visits, :fleet
+  Error           = Class.new(StandardError)
+  InvalidEndpoint = Class.new(Error)
+
+  ENDPOINTS = %i(vrp vrp-long pdp pdp-long)
+
+  @@endpoint = ENDPOINTS.first
```

Comment:
Plus is it desirable to have the value on the class itself at all? Makes the code not thread safe (no idea about the rest of the code base though ;)